docs/hooks/use-text-as-image.md

```diff
@@ -73,14 +73,15 @@ It also can handle inserting arbitrary html instead of just text strings.
 
 ## API
 ```js
-const {insertTextAsImage} = useTextAsImage({processSerializedElement})
+const {insertTextAsImage} = useTextAsImage({processSerializedElement, fontFamily: 'Courier New'})
```

Comment:
Usually in this section of the docs, we just put the API, not example code.

So instead of putting Courier New, you just put this:
```js
const {insertTextAsImage} = useTextAsImage({processSerializedElement, fontFamily})
```